src/config/Config.zig

```diff
@@ -2309,6 +2309,30 @@ term: []const u8 = "xterm-ghostty",
 /// This only works on macOS since only macOS has an auto-update feature.
 @"auto-update-channel": ?build_config.ReleaseChannel = null,
 
+/// Bell features to enable if bell support is available in your runtime. The
+/// format of this is a list of features to enable separated by commas. If you
+/// prefix a feature with `no-` then it is disabled. If you omit a feature, its
+/// default value is used, so you must explicitly disable features you don't
+/// want.
+///
+/// Available features:
+///
+///   * `system` - Use a system function to play an audible sound. This differs
+///     from the `audio` feature in that the sound played is not customizable
+///     from within Ghostty. Your system may allow for the sound to be
+///     customized externally to Ghostty.
+///   * `audio` - Play a custom sound. (GTK only).
+///
+/// Example: `audio`, `no-audio`, `system`, `no-system`:
+///
+/// By default, no bell features are enabled.
+@"bell-features": BellFeatures = .{},
+
+/// If `audio` is an enabled bell feature, this is a path to an audio file.
+/// If the path is not absolute, it is considered relative to the `media`
+/// subdirectory of the Ghostty configuration directory.
```

Comment:
I'm really not sure whether we should do this. I think people should just always specify absolute paths so that it works regardless of Ghostty configs

Reply:
How does it work for `config-file` or `custom-shader`? IIRC in both cases a relative path resolves relative to the directory of the config file, which seems reasonable to do here too

Reply:
Hmm it's all a bit convoluted but it looks like `config-file`, `custom-shader`, and `gtk-custom-css` are always treated relative to the directory of the file that they are referenced from, or in the case of CLI args, the current working directory. Re-using that logic I think would be fine.

Reply:
I pushed PR #6622 to refactor RepeatablePath and add NonRepeatablePath (because I don't think it makes any sense to have multiple audio files). This way `bell-audio` has the same behavior as other configuration entries with regards to relative directories or expanding `~/`.